rust/otap-dataflow/crates/otap/src/syslog_cef_receiver/parser/parsed_message.rs

```diff
@@ -9,30 +9,30 @@ use otel_arrow_rust::encode::record::attributes::AttributesRecordBatchBuilder;
 use std::borrow::Cow;
 
 // Common attribute key constants for both RFC5424 and RFC3164 messages
-const SYSLOG_FACILITY: &str = "syslog.facility";
-const SYSLOG_SEVERITY: &str = "syslog.severity";
-const SYSLOG_HOST_NAME: &str = "syslog.host_name";
-const SYSLOG_MESSAGE: &str = "syslog.message";
+const SYSLOG_FACILITY: &[u8] = b"syslog.facility";
```

Comment:
These were const str slices which were already UTF-8 validated. I might be wrong here, but it seems like by updating them to bytes slices we would end up having to UTF-8 validate them each time we build the attributes record batch.